pyupgrade/_plugins/native_literals.py

```diff
@@ -58,4 +71,4 @@ def visit_Call(
             state.settings.min_version >= (3,) and
             is_a_native_literal_call(node, state.from_imports)
```

Comment:
this function name and implementation no longer matches -- I would do a separate condition entirely for bytes

Reply:
I added a `is_a_bytes_from_bytestr_call` check. I feel like splitting up the `_fix_native_*` function does not make too much sense, since both would only be different by one character (`src="b''"`) or is adding the `if` a big overhead?